task.2.py

```diff
@@ -0,0 +1,24 @@
+# solution for tasks two
+cloak = input()
+print(cloak)
+
+print(f'Сколько времени?{cloak}')
+
+with open('text.txt', 'w') as f:
+    print(f.write('hello_world'))
+def calculator (a,b):
+    return (a+b)
+print(calculator(1, 3))
+
+def total (*numbers):
```

Comment:
The declare of func is correct, the next lines are not. Function is not perform the idea it have to do